modules/tide_jira/src/TideJiraAPI.php

```diff
@@ -216,9 +235,59 @@ private function getRevisionInfo(NodeInterface $node) {
       'is_new' => $node->isNew() ? 'New page' : 'Content update',
       'notes' => $node->getRevisionLogMessage(),
       'preview_links' => $this->getPreviewLinks($node, TRUE),
+      'site' => $this->entityTypeManager->getStorage('taxonomy_term')->load($node->get('field_node_primary_site')->getValue()[0]['target_id'])->getName(),
+      'site_section' => $this->getSiteSections($node),
+      'page_department' => $this->entityTypeManager->getStorage('taxonomy_term')->load($node->get('field_department_agency')->first()->getValue()['target_id'])->getName(),
```

Comment:
Does it need a check first if this exist and then get the value !

Reply:
Hi @MdNadimHossain,

As discussed, these fields are required on content-vic and should have a value.

If they're not available (i.e if deployed on another project), we can't just pass an empty string as this would affect reporting.

I've added further detail to the README to detail setup of these required fields.